controllers/duration.js

```diff
@@ -1,10 +1,28 @@
+<<<<<<< HEAD
+const logDuration = require('../models/duration');
+=======
 // const log = require('../models/duration');
+>>>>>>> main
 
 // module.exports = {
 //   index,
 // };
 
+<<<<<<< HEAD
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7840230)
```suggestion
} 
```